core/storage/suggestion/gae_models.py

```diff
@@ -610,3 +613,83 @@ def export_data(cls, user_id):
                 'rejection_message': voiceover_model.rejection_message
             }
         return user_data
+
+
+class ReviewerAndSuggestionCountsModel(base_models.BaseModel):
```

Comment:
I see your point but I'm not sure about `SuggestionReviewerAggregateStatsModel` because it could be a little misleading that the model is just about suggestion reviewers and not suggestions and reviewers separately. Also, I'm not sure we would ever store stats - wouldn't it be better to store the raw data and then just have a helper method in the domain class to calculate the stat that you want, that way the stats are always up to date/inconsistencies are less likely? That's what I'm going to implement for the ratios.